templates/async.yaml.erb

```diff
@@ -1,18 +1,18 @@
-async: !ruby/object:Api::Async
```

Comment:
Not sure, this was just touched because of the global rename. If nothing breaks when it's gone I can remove. cc @rambleraptor in case he can answer faster.

Reply:
I feel like this is safe to be deleted. api.yaml doesn't have nested compilation anymore, which is what this file was used for.